lib/specs/v2.js

```diff
@@ -455,45 +455,45 @@ let rules = {
     },
     'GS050-CSS-KGWW': {
         level: 'error',
-        rule: 'The <code>.kg-width-wide</code> CSS class is required to use your theme with 2.0',
-        details: oneLineTrim`The <code>.kg-width-wide</code> CSS class is required to use your theme with 2.0.
-        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/" target=_blank>here</a>.`,
+        rule: 'The <code>.kg-width-wide</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-width-wide</code> CSS class is required otherwise the wide images will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#gallery-card" target=_blank>here</a>.`,
         regex: /\.kg-width-wide/g,
         className: '.kg-width-wide',
         css: true
     },
     'GS050-CSS-KGWF': {
         level: 'error',
-        rule: 'The <code>.kg-width-full</code> CSS class is required to use your theme with 2.0',
-        details: oneLineTrim`The <code>.kg-width-full</code>  CSS class is required to use your theme with 2.0.
-        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/" target=_blank>here</a>.`,
+        rule: 'The <code>.kg-width-full</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-width-full</code> CSS class is required otherwise the full width images will appear unstyled.
+        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/#gallery-card" target=_blank>here</a>.`,
         regex: /\.kg-width-full/g,
         className: '.kg-width-full',
         css: true
     },
     'GS050-CSS-KGGC': {
         level: 'error',
-        rule: 'The <code>.kg-gallery-container</code> CSS class is required to use your theme with 2.0',
-        details: oneLineTrim`The <code>.kg-gallery-container</code>  CSS class is required to use your theme with 2.0.
-        Find out more about required theme changes for the Koenig editor <a href="${docsBaseUrl}editor/" target=_blank>here</a>.`,
+        rule: 'The <code>.kg-gallery-container</code> CSS class is required to appear styled in your theme',
+        details: oneLineTrim`The <code>.kg-gallery-container</code>  CSS class is required otherwise the gallery images will appear unstyled.
```

Comment:
```suggestion
        details: oneLineTrim`The <code>.kg-gallery-container</code>  CSS class is required otherwise galleries will appear unstyled.
```